contracts/ContractRegistryAccessor.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.5.16;
+
+import "@openzeppelin/contracts/ownership/Ownable.sol";
+import "./spec_interfaces/IContractRegistry.sol";
+import "./spec_interfaces/IProtocol.sol";
+import "./spec_interfaces/IFees.sol";
+import "./spec_interfaces/ICommittee.sol";
+import "./interfaces/IElections.sol";
+import "./spec_interfaces/IValidatorsRegistration.sol";
+import "./spec_interfaces/ICompliance.sol";
+import "./spec_interfaces/ISubscriptions.sol";
+
+contract ContractRegistryAccessor is Ownable {
```

Comment:
is there an easy way to share constants by more than one contract? we probably should not do it by cross contract calls (getters/public constant vars) for gas reasons. but a library might do the trick?...

a possible //TODO?

Reply:
Which constants are you referring to?

Reply:
created issue, resolving